pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -628,22 +695,34 @@ def _set_dimensionless_parameters(self):
         self.T_init = self.therm.T_init
         self.c_e_init = self.c_e_init_dimensional / self.c_e_typ
 
-    def chi(self, c_e):
+        # Dimensionless mechanical parameters
+        if isinstance(self, pybamm.particle_cracking.BaseCracking):
+            self.rho_cr_n = self.rho_cr_n_dim * self.l_cr_n_0 * self.w_cr
+            self.rho_cr_p = self.rho_cr_p_dim * self.l_cr_p_0 * self.w_cr
+            self.theta_p = self.theta_p_dim * self.c_p_max / self.T_ref
+            self.theta_n = self.theta_n_dim * self.c_n_max / self.T_ref
+            self.t0_cr = 3600 / self.timescale  # nomarlised typical time for one cycle
```

Comment:
should this timescale be set relative to the discharge timescale rather than 1hr? I've not checked how/where it appears in the model (i.e. why not just be 1?)

Reply:
solved below